.github/workflows/ci.yml

```diff
@@ -9,6 +9,23 @@ on:
       - '!main'
 
 jobs:
+  lint:
+    name: Linting with prettier
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
```

Comment:
This is only needed for commit lint to work, so no need to add it here (but see next comment)